package.json

```diff
@@ -80,19 +80,25 @@
     "db-migration:list": "npm run db-migration:base list --"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.15",
```

Comment:
Why do we need these fontawesome dependencies? Fontawesome comes already with phovea_ui and can be used by adding the classes to the element (e.g., `<i class="fas fa-question"></i>`).